src/insight/alembic/versions/a2198ae60b44_added_version_name_unique_constaint.py

```diff
@@ -0,0 +1,27 @@
+"""Added Result.run_id
+
+Revision ID: a2198ae60b44
+Revises: d2198fd60b0e
+Create Date: 2023-12-13 13:25:17.878689
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a2198ae60b44"
+down_revision = "d2198fd60b0e"
```

Comment:
What's this value?

Reply:
Previous revision in a tree

Reply:
Ahh I hadn't pulled the latest version so didn't see that version. Nice 👍